src/ts/components/dashboard/Dashboard.tsx

```diff
@@ -220,9 +224,11 @@ class Dashboard extends React.Component<IDashboardProps, IDashboardState> {
           </Dropdown>
         </Section>
         <Section>
+          <Tooltip title='Copy to clipboard' position='bottom' trigger='mouseenter' arrow={true}>
```

Comment:
One minor thing @hskang9 can you please define 'Copy to clipboard' in `_locales/en/messages.json` file and use i18n service to reference that?

Reply:
okay will do that.

Reply:
done.